src/main/java/prefs/Preferences.java

```diff
@@ -118,6 +118,16 @@ public void addBoard(String name, List<PanelInfo> panels) {
     public void removeBoard(String name) {
         global.removeBoard(name);
     }
+
+    public void setLastDeletedIndex(String deletedBoardName){
```

Comment:
Add header comments. 
This method name is not specific enough as it doesn't mention Boards.